Gemfile

```diff
@@ -10,8 +10,8 @@ def vanagon_location_for(place)
   end
 end
 
-gem 'vanagon', *vanagon_location_for(ENV['VANAGON_LOCATION'] || 'git@github.com:puppetlabs/vanagon#master')
-gem 'packaging', :github => 'puppetlabs/packaging', :branch => 'master'
+gem 'vanagon', *location_for(ENV['VANAGON_LOCATION'] || '~> 0.15')
+gem 'packaging', *location_for(ENV['PACKAGING_LOCATION'] || '~> 0.99')
 gem 'artifactory'
```

Comment:
Can we drop this dependency?

Reply:
Why did we have it initially?

Reply:
because artifactory is required by packaging, but when installing packaging from source we don't pick up its gem dependencies

Reply:
Ooo gotcha, I'll remove